model_compression_toolkit/core/common/quantization/set_node_quantization_config.py

```diff
@@ -205,7 +205,7 @@ def set_quantization_configs_to_node(node: BaseNode,
                 # Preserving the quantization of more than 1 previous node is ambiguous, so disable it.
                 Logger.info(f"Disabling Quantization-Preserving for node {node.name} because it has more than 1 input activations.")
                 candidate_qc.activation_quantization_cfg.quant_mode = ActivationQuantizationMode.NO_QUANT
-            elif not prev_nodes[0].is_quantization_preserving() or not prev_nodes[0].is_activation_quantization_enabled():
```

Comment:
I think that this was already fixed in the main branch, please sync your branch with main so we can verify that there is no override of committed changes